genie/process_functions.py

```diff
@@ -1041,3 +1041,112 @@ def get_gdc_data_dictionary(filetype):
         .format(filetype=filetype))
     gdc_response = json.loads(gdc_dict.text)
     return(gdc_response)
+
+
+def _create_schema(syn, table_name, parentid, columns=None, annotations=None):
+    """Creates Table Schema
+
+    Args:
+        syn: Synapse object
+        table_name: Name of table
+        parentid: Project synapse id
+        columns: Columns of Table
+        annotations: Dictionary of annotations to add
+
+    Returns:
+        Schema
+    """
+    schema = synapseclient.Schema(name=table_name,
+                                  columns=columns,
+                                  parent=parentid)
+    schema.annotations = annotations
+    new_schema = syn.store(schema)
+    return new_schema
+
+
+def _update_database_mapping(syn, database_synid_mappingdf,
+                             database_mapping_synid,
+                             fileformat, new_tableid):
+    """Updates database to synapse id mapping table
+
+    Args:
+        syn: Synapse object
+        database_synid_mappingdf: Database to synapse id mapping dataframe
+        database_mapping_synid: Database to synapse id table id
+        fileformat: File format updated
+        new_tableid: New file format table id
+
+    Returns:
+        Updated Table object
+    """
+    fileformat_ind = database_synid_mappingdf['Database'] == fileformat
+    # Store in the new database synid
+    database_synid_mappingdf['Id'][fileformat_ind] = new_tableid
+    # Only get the vcf2maf row so that only this row is updated in the
+    # mapping table
+    to_update_row = database_synid_mappingdf[fileformat_ind]
+
+    # Update this synid later (This synid needs to not be hardcoded)
+    updated_table = syn.store(synapseclient.Table(database_mapping_synid,
+                                                  to_update_row))
+    return updated_table
+
+
+def _move_entity(syn, ent, parentid, name=None):
```

Comment:
TODO deprecate once some jira for move function is out of the cli into the client

Reply:
Actually the `move` functionality is already part of the `develop` branch.  https://github.com/Sage-Bionetworks/synapsePythonClient/pull/550/files